src/main/java/core/basesyntax/FigType.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum FigType {
```

Comment:
it's only used in switch case - we can do fine without it, just use integers instead, to simplify your codebase